actions/workflows/create_vm.yaml

```diff
@@ -2,17 +2,15 @@
   chain:
     -
       name: "get_subnet_id"
-      ref: "st2cd.kvstore"
+      ref: "st2.kv.get"
```

Comment:
I quickly checked https://github.com/StackStorm/st2puppet/tree/production/hieradata/ and it doesn't seem so, so we probably need to update it if we want to use that pack.